content/news/2021-02-21-week-12-progress.md

```diff
@@ -0,0 +1,14 @@
+title: "About my Outreachy internship with Mixxx DJ: Week 12 progress"
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship, manual
+date: 2021-02-21 21:50:00
+
+_This is the twelfth weekly progress report of my Outreachy internship with Mixxx DJ._
+
+This week:
+
+- I added the step by step tutorial on deck cloning for playing with a single turntable. See [pull request #330](https://github.com/mixxxdj/manual/pull/330) I plan to do the same for beat juggling, but I want to get feedback on how that looks before I proceed to do the same for beat juggling.
+- I made the video tutorial introducing Mixxx to the first time user – a tutorial guided by [chapter 4](https://manual.mixxx.org/2.2/en/chapters/library.html) of the manual. I’m still making edits to the video, but I’m hoping to finish with this by the end of the week.
+- I lost a couple of days to long distance travels – from my home to the city and back. I had also to do bit of adjusting migrating from Windows to MacOS, but I’m getting there 🙂
+- This week (since I’m writing this blog late), I will be making a pull request for [issue #293](https://github.com/mixxxdj/manual/issues/293). “Mention mp3 encoder in Recording chapter”. I had posed a question in the comments section, hopefully I will get an answer this week.
```

Comment:
```suggestion
- This week (since I’m writing this blog late), I will be making a pull request for [issue #293](https://github.com/mixxxdj/manual/issues/293) (“Mention mp3 encoder in Recording chapter”). I posted a question in the comments section, hopefully I will get an answer this week.
```